src/common/sorted-table/index.js

```diff
@@ -222,6 +222,31 @@ class IndividualAction extends Component {
   }
 }
 
+class GroupedAction extends Component {
+  _getIsDisabled = createSelector(
+    () => this.props.disabled,
+    () => this.props.selectedItems,
+    () => this.props.userData,
+    (disabled, selectedItems, userData) =>
+      isFunction(disabled) ? disabled(selectedItems, userData) : disabled
```

Comment:
I think you also need to check if all the items are selected (ie `this.state.all` is true) because in that case, you need to pass all the items to the `disabled` function.

Reply:
I added condition to `selectedItems`